nservicebus/recoverability/subscribing-to-error-notifications.md

```diff
@@ -18,6 +18,14 @@ Error notifications are available for several events:
  * When a [delayed retry](/nservicebus/recoverability/#delayed-retries) occurs.
  * When a message fails all retries and is forwarded to the error queue.
 
+## Satellite behavior
+
+Since Version 6 errors happening in satellites will not trigger error notification and requires a [custom retry policy configuration per satellite](/nservicebus/satellites/#implementing-a-satellite-managing-errors).
+
+NOTE: The Gateway is a satellite see the [Gateway Version 1 to 2 upgrade guide](/nservicebus/upgrades/gateway-1to2.md) for more information.
```

Comment:
Does it make sense to link to the GW here? Feels off to me, there are other satellites and we don't link to them?